src/lake/test/116/test.sh

```diff
@@ -14,14 +14,7 @@ INITD_NOT=$'Content-Length: 40\r\n\r\n{"jsonrpc":"2.0","method":"initialized"}'
 OPEN_REQ=$'Content-Length: 145\r\n\r\n{"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file://Test.lean","languageId":"lean4","version":0,"text":""}}}'
 MSGS="$INIT_REQ$INITD_NOT$OPEN_REQ"
 
-set -o pipefail
-
-cp /dev/null lake.in
-tail -f lake.in | $LAKE serve 2>&1 > lake.out &
-pid=$!
-echo -n "$MSGS" >> lake.in
-sleep 3
-kill $pid
+(echo -n "$MSGS"; sleep 3) | $LAKE serve 2>&1 > lake.out || true
```

Comment:
Curiously this broke under Nix, perhaps because Nix' isolation interferes with sending the signal to the process group. A more proper solution would be to send the correct shutdown notification to the process

Reply:
I am pretty sure sending that, for some reason, the shutdown notification was insufficient when I was writing this test. I should have made a note of why, though, because I have since forgotten.

Reply:
Happy with my version for now :smile: